connectors/v2/subphonic.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#playdeck';
+
+Connector.getTrackArt = function() {
+	return $('#coverartimage img').attr('src') || null;
+};
+
+Connector.getArtist = function() {
+	return $('#songdetails_artist').text().split(' - ')[0] || null;
+}
+
+Connector.trackSelector = '#songdetails_song';
+
+Connector.isPlaying = function() {
+	return !$('#playdeck audio').paused
+}
+
+Connector.getAlbum = function() {
+	return $('#songdetails_artist').text().split(' - ')[1] || null;
+}
+
+Connector.currentTimeSelector = '#played';
+
+Connector.durationSelector = '#duration';
+
+Connector.playButtonSelector = '#PlayTrack'
```

Comment:
If you overwrite the isPlaying function, playButtonSelector isn't used. You can remove this line.
https://github.com/david-sabata/web-scrobbler/blob/master/core/content/connector.js#L221